### PR TITLE
mutt: new package at 1.12.2

### PR DESCRIPTION
**Pull request by penguin-ff**
## Description of package

The Mutt E-Mail Client

## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

**Comment by penguin-ff** [comment]
I decided for mutt as email client. The files are based on your yesterday's boilerplate :-)

File: community/mutt/build [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -e
+
+./configure \
+    --prefix=/usr \
+    --with-screen=ncurses \
+    --sysconfdir=/etc \
+    --disable-nls
```
Comment (dylanaraps) [inline community/mutt/build]: Does it build without `--disable-nls`?

File: community/mutt/build [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -e
+
+./configure \
+    --prefix=/usr \
+    --with-screen=ncurses \
+    --sysconfdir=/etc \
```
Comment (dylanaraps) [inline community/mutt/build]: Put `--sysconfdir` before `--with-screen`.

File: community/mutt/depends [hunk 3]
```diff
@@ -0,0 +1,3 @@
+autoconf make
```
Comment (dylanaraps) [inline community/mutt/depends]: Is `autoconf` needed?

File: community/mutt/depends [hunk 4]
```diff
@@ -0,0 +1,3 @@
...
+automake make
```
Comment (dylanaraps) [inline community/mutt/depends]: Is `automake` needed?

File: community/mutt/depends (hunk 3 above)
Comment (penguin-ff) [reply community/mutt/depends]: no - sorry...

File: community/mutt/depends (hunk 4 above)
Comment (penguin-ff) [reply community/mutt/depends]: not needed either...

File: community/mutt/build (hunk 1 above)
Comment (penguin-ff) [reply community/mutt/build]: Yes. I removed it...

File: community/mutt/build [hunk 5]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -e
+
+./configure \
+    --prefix=/usr \
+    --sysconfdir=/etc \
+    --with-screen=ncurses
```
Comment (dylanaraps) [inline community/mutt/build]: It's worth testing without this line too. :)

File: community/mutt/build (hunk 2 above)
Comment (penguin-ff) [reply community/mutt/build]: Done ! :-)

File: community/mutt/build (hunk 5 above)
Comment (penguin-ff) [reply community/mutt/build]: The "ncurses line" ?

Comment (dylanaraps) [reply community/mutt/build]: Yeah.

Comment (penguin-ff) [reply community/mutt/build]: Done! I realize, you are a real specialist :-)

**Comment by dylanaraps** [comment]
Looks good! :+1:

Let me know if it's ready to merge. :)

**Comment by penguin-ff** [comment]
I'll do some testing first... Who's job is it to click on the "resolve conversation button" here in github? Yours or is it mine?

**Comment by dylanaraps** [comment]
Don't worry about the resolve button, I don't really use it. So long as we're both happy with the PR, I'll push the big green merge button!

**Comment by penguin-ff** [comment]
For sending emails via SMTP / SSL i need to create and use package "cyrus-sasl" 
What Du you suggest: Create separate pull request for the new package or shall i mix it in this pull request / these commits?

**Comment by dylanaraps** [comment]
A separate pull request. :+1:

**Comment by penguin-ff** [comment]
So i suppose i'd insert the pull request for the cyrus-sasl and after that it's finished, i continue with this one... OK?

**Comment by dylanaraps** [comment]
Open the other pull request, I'll review and merge and finally merge this one. :+1:

**Comment by dylanaraps** [comment]
`cyrus-sasl` needs to be in a **new** pull request and not this one.

**Comment by penguin-ff** [comment]
Mutt / cyrus-sasl now working fine for me. Accidentally only on my "test-box" i got a: "conflicts with baselayout" because of /etc/mime.types - but i suppose baselayout is not installed by standard - at least not on my 1st machine. So what's the correct way to deal with this problem?
BTW: I klicked "new pull request" but after that the button "create pull request" was gone - Do you know what went wrong?

**Comment by dylanaraps** [comment]
To open a separate pull request, `cyrus-sasl` needs to be in a *different* `git branch` to `mutt`. The `baselayout` package is always installed in KISS and if there is a conflict the file should be removed from whatever package is additionally providing it.

In the package's `build` script: `rm -f "$1/etc/mime.types"`.

File: community/mutt/depends [hunk 6]
```diff
@@ -0,0 +1,4 @@
+cyrus-sasl
+libressl
```
Comment (dylanaraps) [inline community/mutt/depends]: This can be removed as `cyrus-sasl` depends on `libressl` which pulls it in.

**Comment by dylanaraps** [comment]
One tiny change (*removing `libressl` from `depends`*) and I'll merge! :)

**Comment by dylanaraps** [comment]
LGTM :+1:

**Comment by dylanaraps** [comment]
Thanks!

**Pull request merged**